### DISCUSSION TITLE
Get data from sd card deck

### DISCUSSION THREAD

**Discussion by minisol0175** [bitcraze/discussions#724] [Q&A]
Hello, I have question about SD card deck.

When getting log data, I can use cfusdlog.py and example.py.
Here, the log is stored in the form of a dictionary.
I want to acquire only some data of this.

However, the exact data does not come out when extracted.
![Screenshot from 2023-05-12 15-41-51](https://github.com/bitcraze/discussions/assets/42127467/edbe8680-3b72-446f-87dc-c42ff815af69)

All data should appear, but it is omitted with "...".
![Screenshot from 2023-05-12 15-41-42](https://github.com/bitcraze/discussions/assets/42127467/750b7719-5337-42b8-90b3-9114d10c0052)

I want to solve this...

config.txt is here.
![Screenshot from 2023-05-12 15-38-06](https://github.com/bitcraze/discussions/assets/42127467/738ce6fa-7f6d-428e-b0d3-056bb84691e2)

**Comment by tobbeanton** [comment]
I think you are only writing the dictionaries printable summary to the file. What about writing it to a cvs file like [this](https://www.tutorialspoint.com/How-to-save-a-Python-Dictionary-to-CSV-file#:~:text=Easiest%20way%20is%20to%20open,pair%20in%20comma%20separated%20form.&text=The%20csv%20module%20contains%20DictWriter,list%20object%20containing%20field%20names.) example.
